modules/storage-api/src/main/java/org/apache/ignite/internal/storage/engine/StorageEngine.java

```diff
@@ -78,4 +80,12 @@ static long defaultDataRegionSize() {
         //noinspection NumericCastThatLosesPrecision
         return Math.max(256 * MiB, (long) (0.2 * getTotalMemoryAvailable()));
     }
+
+    /**
+     * Returns IDs of tables for which there are MV partition storages that were created, but they were not yet destroyed
+     * (or the destruction did not finish).
+     */
+    default Set<Integer> nonDestroyedTableIds() {
```

Comment:
Does it really make sense to use word "destroy" in this context? I feel like the name would be simpler if we called it something like `getAllTableIds`, because that's what it does. Is this correct?

Reply:
It's subtle: there are tables contained in the storage engines (for which createMvTable() was called in this Ignite instance), and there are data structures on disk that represent tables in the storage engine. Those are different. This difference is especially visible for volatile storage engines: they can have some tables, but they never have anything on disk, so the new method I added will always return an empty set for them.

How about `tableIdsOnDisk()`?

Reply:
That's more obvious, I like it, thank you!